sources/packages/backend/apps/workers/src/controllers/disbursement/disbursement.controller.ts

```diff
@@ -64,4 +73,38 @@ export class DisbursementController {
       );
     }
   }
+
+  /**
+   * Associates an MSFAA number to the disbursement(s) checking
+   * whatever is needed to create a new MSFAA or use an
+   * existing one instead.
+   */
+  @ZeebeWorker(Workers.AssociateMSFAA, {
+    fetchVariable: [ASSESSMENT_ID],
+    maxJobsToActivate: MaxJobsToActivate.Low,
+  })
+  async associateMSFAA(
+    job: Readonly<
+      ZeebeJob<AssignMSFAAJobInDTO, ICustomHeaders, IOutputVariables>
```

Comment:
Since the worker was moved, is it not the case to move `AssignMSFAAJobInDTO` also?

Reply:
Yes. Nice catch.